src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -846,7 +847,37 @@ public PdaoLoadStatistics loadToStagingTable(
           (ingestRequest.getCsvNullMarker() == null) ? "" : ingestRequest.getCsvNullMarker());
     }
     LoadJobConfiguration configuration = loadBuilder.build();
```

Comment:
If this is only used in the `else` on line 860, it can be moved inside that block to make its scope smaller.